language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionUtil.java

```diff
@@ -183,4 +201,121 @@ public static CodeActionNodeType topLevelNodeInLine(LSContext context, TextDocum
 
         return lsDiagnostics;
     }
+
+    /**
+     * Returns an object that is included with codeStart, bodyStart and bodyEnd positions of a given node.
+     *
+     * @param node Node for which the positions are calculated
+     * @return {@link NodePositions} of the given node
+     */
+    private static NodePositions getNodeSelectionPositions(Node node) {
+        if (!(node instanceof NonTerminalNode)) {
+            return new NodePositions(null, null, null);
+        }
+        NonTerminalNode member = (NonTerminalNode) node;
+        int codeStartLine = member.lineRange().startLine().line() + getNumberOfDocLines(member);
+        Position startCodePosition = new Position(codeStartLine, member.lineRange().startLine().offset());
+
+        switch (member.kind().name()) {
+            case "FUNCTION_DEFINITION":
```

Comment:
kind is an enum here. So you can directly use the kinds in the cases instead of the string

Reply:
Updated.